app/Traits/HasRole.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Traits;
+
+trait HasRole
+{
+    public function hasRole($role)
+    {
+        if (is_string($role)) {
+            return $this->role->name === $role ? true : false;
```

Comment:
viết gọn lại là $this->role->name === $role được rồi e, khong cần true false phía sau